app/views/posts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+    <h1>Add a new post</h1>
+    <% form_with model:@posts, url: user_posts_path, class: "form" do |f| %>
+        <%= f.label :title %><br>
+        <%= f.text_field :title %><br>
+        <%= f.label :text %><br>
+        <%= f.text_area :text, cols: 30, rows: 10 %><br>
+        <%= f.submit "Submit" %>
+    <% end %>
+</div>
```

Comment:
- [ ] I can't see the post form on the page. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/179086410-14095fec-4d71-4060-81ee-71c192f463db.png)